pages/common/grep.md

```diff
@@ -3,34 +3,38 @@
 > Matches patterns in input text.
 > Supports simple patterns and regular expressions.
 
+- Search for a pattern:
+
+`grep {{^regex$}} {{path/to/file}}`
+
 - Search for an exact string:
 
-`grep {{search_string}} {{path/to/file}}`
+`grep -F {{search_string}} {{path/to/file}}`
 
 - Search in case-insensitive mode:
 
-`grep -i {{search_string}} {{path/to/file}}`
+`grep -i {{^regex$}} {{path/to/file}}`
```

Comment:
If you put `^` and `$` here, I think it only filter the line exactly match the `regex` here.

Reply:
I don't think it's actually necessary to replace all `search_string`s with regexes. That way someone could get a wrong impression that you must use a regex with grep.

Reply:
Is this how regexes were displayed inside `{{}}` in the other pages?